site/_posts/2020-05-20-CSI-integration.md

```diff
@@ -0,0 +1,202 @@
+---
+title: "CSI Volume Snapshotting Capabilities in Velero"
+excerpt: Velero 1.4 now ships with support for CSI snapshotting v1beta1 APIs.
+author_name: Ashish Amarnath
+categories: ['velero','??']
+image: ??
```

Comment:
Not sure what categories mean.
Also, don't know what image to use, and should we use an image.

Reply:
I don't think we're currently using the categories for anything, presumably it's for a tag cloud/similar.

You get to pick the image! https://unsplash.com/ is a good source for free ones.